201-web-app-sql-database/azuredeploy.json

```diff
@@ -49,7 +68,21 @@
       "metadata": {
         "description": "Location for all resources."
       }
-    }
+    },
+    "actionGroupId": {
+      "type": "string",
+      "defaultValue": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourcegroup().name, '/providers/Microsoft.insights/actiongroups/', parameters('ActionGroupName'))]",
```

Comment:
remove concat and use resourceId()

Reply:
@bmoore-msft thanks considering metric issue at present as failure seems to associated to upgrading from Appinsights.